share/spice/people_in_space/people_in_space.js

```diff
@@ -17,6 +17,29 @@
             };
             obj.templates.item = 'base_item';
             obj.meta.itemType = number === 1 ? "Person in Space" : "People in Space";
+            
+            obj.onItemShown = function(item) {
+                // Get filename from biophoto image url. Replace file extention jpg to png
+                // Use DDG.get_asset_path to get full path to images under spice directory
+                var imageFilename = item.biophoto.replace(/^.*[\\\/]/, ''),
+                    localImage = DDG.get_asset_path('people_in_space', "images/" + imageFilename)
+                
+                // Perform a HTTP GET request to retrieve local images using filename from API 
+                // This is used to determine if a local image exists 
+                $.ajax({
```

Comment:
This might be slightly simpler? I haven't tested it though...

```js
$.get(localImage, function(data, statusText, xhr) {
  item.set('image', localImage);
}, "image/jpeg")
.error(function() {
  item.set('image', item.biophoto);
});
```